packages/core/sdk/src/services/internal/CoreStitchServiceClientImpl.ts

```diff
@@ -70,9 +70,9 @@ export default class CoreStitchServiceClientImpl
   ): StitchAuthRequest {
     const body = { name };
     if (this.serviceName !== undefined) {
-      body["service"] = this.serviceName;
+      body["service".toString()] = this.serviceName;
```

Comment:
[nit] this seems weird, why is this necessary? could we perhaps improve this by making the type better?